config/routes.rb

```diff
@@ -8,4 +8,5 @@
   root "home#index"
   
   get "/health_check", to: 'health_check#health_check', as: :rails_health_check
+  get "/search_stats", to: 'search_stats#index', as: :stats_stats_list
```

Comment:
We don't really need this option, or if it's your intention, please share it with us. 🙏 

```suggestion
  get "/search_stats", to: 'search_stats#index'
```